templates/sandbox-checkout-alert.php

```diff
@@ -0,0 +1,15 @@
+<?php
+if ( ! $is_sandbox_mode ) {
+	return;
+}
+
+$messages = array(
```

Comment:
You should pass the localized message on `wc_get_template`
See [templates/checkout-conversion-rate.php](https://github.com/ebanx/woocommerce-gateway-ebanx/blob/master/templates/checkout-conversion-rate.php) and [gateways/class-wc-ebanx-gateway.php:1351](https://github.com/ebanx/woocommerce-gateway-ebanx/blob/master/gateways/class-wc-ebanx-gateway.php#L1351)